README.md

```diff
@@ -8,14 +8,28 @@ Betterbird is a fine-tuned version of [Mozilla Thunderbird](https://www.thunderb
 Non flatpak-packaging issues should be reported upstream at<br>
 https://bugzilla.mozilla.org/describecomponents.cgi?product=Thunderbird
 
-#### Migration from pre-exisiting non-flatpak installations
-In order to migrate from pre-exisiting non-flatpak installation and preserve all settings please copy or move entire<br>
+#### Migration from pre-exisiting installations
+
+##### Migration from pre-exisiting Thunderbird flatpak installations
+In order to migrate from pre-exisiting Thunderbird flatpak installation and preserve all settings please copy or move entire<br>
+`~/.var/app/org.mozilla.Thunderbird/.thunderbird`<br>
+folder into<br>
+`~/.var/app/eu.betterbird.Betterbird/.betterbird`
```

Comment:
This should be `~/.var/app/eu.betterbird.Betterbird/.thunderbird`. Betterbird does not find the profile if it is in `.betterbird` folder.

Reply:
Done